.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: go
 
 go:
-  - "1.10"
-  - "1.11"
-  - "1.12"
+  - "1.12.5"
```

Comment:
Can we use 1.12 here?

Reply:
I followed kubefed here https://github.com/kubernetes-sigs/kubefed/pull/902 , it is always good to use latest stable version.

Reply:
Personally, I do not think so. I think we should support multiple go versions.

Reply:
Different go version sometimes have different standard. In my previous PR, I found that `gofmt` always failed in go1.10 but succeed in go 1.12 and go 1.11, this caused the `make check` failed. Using the latest go can leverage the latest features in Go.

Reply:
Yeah, go 1.10 can be removed. Format check may be failed in 1.10. I also noticed it.